orc8r/protos/bootstrapper.proto

```diff
@@ -71,3 +73,114 @@ service Bootstrapper {
   // Returns signed certificate.
   rpc RequestSign (Response) returns (Certificate) {}
 }
+
+// --------------------------------------------------------------------------
+// Protos for CloudRegistration and Registration servicer
+// --------------------------------------------------------------------------
+/**
+// Operator Side Registration
+Operator->REST API:
+  GET_gateway(network_id, logical_id)
+
+  REST API<->CloudRegistration:
+    GetToken(network_id, logical_id, refresh): nonce, timeout
+
+  REST API<->CloudRegistration:
+    GetInfoForGatewayRegistration(): domain_name, root_ca
+
+Operator<-REST API:
+  GatewaySwaggerObject{..., registration_field{nonce, timeout, domain_name, root_ca}}
+
+// AGW Side Registration
+Operator->AGW:
+  passes nonce, domain_name, root_ca
+
+AGW->Registration:
+  Register(nonce, hwid, challenge_key)
+
+  Registration<->CloudRegistration:
+    GetGatewayRegistrationInfo(nonce): network_id, logical_id
+
+  Registration->Device:
+    RegisterDevices(network_id, hwid)
+
+Registration->AGW:control_proxy
+ */
+
+service CloudRegistration {
```

Comment:
To Andrei's point: let's add doc comments directly to the `service` definitions here, same as we do for the endpoints within the service